README.md

```diff
@@ -1,44 +1,268 @@
+
+*This tutorial has been adapted from Erika Pauwels's mu.semte.ch article. You can view it [here](https://mu.semte.ch/2017/04/13/building-your-first-microservice-with-sinatra/)*
+
+
+### Building an RDFa importer service
+![](http://mu.semte.ch/wp-content/uploads/2017/07/IMG_6145-1024x768.png)
+
+RDFa is a way to embed a Semantic Model into Linked Data.  In this tutorial we describe how we can implement a microservice to import these contents into the mu.semte.ch stack.  We will go through our own development process and see what we discovered.
+
+#### Getting started
+
+We create a new folder with the basic stub for a new Ruby template.
+
+```rb
+# /path/to/importer/web.rb
+get '/' do
+ content_type 'application/json'
+ { data: { attributes: { hello: 'world' } } }.to_json
+end
+```
+
+```Dockerfile
+# /path/to/importer/Dockerfile
+FROM semtech/mu-ruby-template:2.4.0-ruby2.3
+MAINTAINER Aad Versteden <madnificent@gmail.com>
+# see https://github.com/mu-semtech/mu-ruby-template for more info
+```
+
+With these files in place we can wire this new service up in a standard mu-project by updating our `docker-compose.yml` and `dispatcher.ex`.
+
+In the `docker-compose.yml` we add our development component and link it to the dispatcher.
+```yaml
+dispatcher:
+  ...
+  links:
+    - rdfaimporter:rdfaimporter
+...
+rdfaimporter:
+  image: semtech/mu-ruby-template:2.4.0-ruby2.3
+  links:
+    - db:database
+  ports:
+    - "8888:80"
+  environment:
+    RACK_ENV: "development"
+  volumes:
+   - "/path/to/importer/:/app"
+```
+
+In the dispatcher, we add the following above *`match _ do`*
+
+```ex
+# new content in dispatcher.ex
+match "/import/*path" do
+  Proxy.forward conn, path, "http://rdfaimporter/"
+end
+```
+
+After starting our project, we can surf to http://localhost/importer and we will receive our default hello world output.  As we update the code in our ruby-template, we will see the updates appear live.
+
+#### Importing RDFa
+
+*Hint: If you’re working your way through this tutorial, you won’t need to execute the steps in this section.*
+
+During our search for a good solution, we search online for a good RDFa importing library.  We find the [rdf-rdfa](https://github.com/ruby-rdf/rdf-rdfa) library on GitHub.  This library looks clean so we create a new Gemfile and add the latest version to it.
+
+```gemfile
+# /path/to/importer/Gemfile
+gem 'rdf-rdfa', '2.2.2'
+```
+
+When we restart the container, which we need to do because we changed the dependencies, we notice that the mu-ruby-template we see the following output.
```

Comment:
to check, quite sure this info is outdated. The image no longer provides rdf-rdfa